.changeset/curvy-penguins-act.md

```diff
@@ -0,0 +1,11 @@
+---
+'@astrojs/tailwind': major
+---
+
+Drops support for Astro <= 5.0 and Tailwind <= 4.0
+
+Tailwind CSS now offers a Vite plugin which is the preferred way to use Tailwind 4 in Astro. This Astro integration is now provided only as a convenience for existing projects if you're unable to upgrade to the Vite plugin at this time. You may continue to use this integration until it is removed.
+
+However, we recommend that you uninstall this integration as soon as you are able. The `astro add tailwind` command will now set up the Vite plugin for you and will no longer install this integration. Or, you can follow the [Tailwind documentation for manual installation](https://tailwindcss.com/docs/installation/framework-guides/astro).
+
+Note the `nesting` option is removed as it's supported out of the box.
```

Comment:
Can we show a code example of this option being removed in a diff? Like our normal "what should I do?" advice?

Otherwise, looks great!

Reply:
I was lazy on that one

``````suggestion
Note the `nesting` option is removed as it's supported out of the box in Tailwind 4.0:

```diff
import { defineConfig } from 'astro/config'
import tailwind from '@astrojs/tailwind'

export default defineConfig({
	integrations: [tailwind({
-		nesting: true
	})]
})
```
``````